frontend/src/interfaces/ApiData.d.ts

```diff
@@ -6,6 +6,8 @@ export type Uuid = string
 export interface Event {
   id: Uuid
   time: number
+  // eslint-disable-next-line camelcase
+  event_type: string
```

Comment:
you can still have your own name here since you copy it in anyways so it doesn't need to be the exact name